Player.py

```diff
@@ -0,0 +1,106 @@
+import random
+
+class Player:
+    """
+    Player class that controls the behaviour of an indivdual player of the game.
+    Contains attributes such as the player name and number of VP'S.
+
+    Attributes:
+    
+    - name : str
+        String representing the name of the player
+    - victory_points: int
+        Integer representing the number of victory points the player has received.
+    - color: tuple
+        A tuple with an r,g,b value representing the color that the player has.
+    - resources: list
+        A list of resources that the player has (all of type Resource)
+    - settlements: list
+        A list of settlements the player has
+    - cities: list
+        A list of cities the player has
+    - roads: list
+        a list of roads the player has (all of type Road)
+    - isTurn: boolean
+        a boolean indicating if it is the players turn in the gameloop.
+    - hasLongestRoad: boolean
+        a boolean indicating if the player has the longest road.
+    - hasLargestArmy: boolean
+        a boolean indicating if the player has the largest army
+    - tradeOffers: list
+        a list of tradeOffers that the player has received (of type Trade)
+
+
+    Methods:
+
+    - roll_dice(self)
+        method for the player to the roll dice on their turn
+    - build_settlement(self, location)
+        builds a settlement at the specified location
+    - build_road(self, location1, location2)
+        builds a road from location1 to location2
+    - buy_card(self)
+        buys card from the bank
+    - play_card(self, card)
+        uses the selected card
+    - make_trade(self, resource, player)
+        activates a trade with the specified player offering the specified Resource
+    - end_turn(self)
+        ends player's turn
+    - get_resources(self)
+        returns list of resources player has
+    - get_victory_points(self)
+        returns number of victory points player has
+    - get_longest_road(self):
+        returns the longest road that the player currently has
+    
+    """
+
+    def __init__(self, name):
+        self.name = name
+        self.victory_points = 0
+        # TODO add implementation for self.color initialisation
```

Comment:
What's the is the self.color going to be?

Reply:
was thinking if we have the GUI to have a section for players and their scores then we can have a colour for each player. Not necessary but might improve the look